src/main/java/com/infamous/combat_next/config/GeneralCombatConfigs.java

```diff
@@ -37,6 +39,16 @@ Adjusts the time in ticks (1/20 seconds) it takes to consume a drink when used.
                             For vanilla, this value is false.
                             """)
                     .define("impaling_change", true);
+            hitboxAdjustmentType = b
+                    .comment("""
+                            Which type of hitbox adjustment should be used.
+                            Base CN hitbox increase expands all dimensions to the minimum value.
+                            CTS hitbox increase only applies if the largest dimension of the entity is less than the minimum.
+                            If CTS does apply to the entity, it inflates all dimensions by the value (minimum - largestDim).
+                            For vanilla, this value is OFF.
+                            For CTS, this value is CTS.
+                            """)
+                    .defineEnum("hitbox_adjustment_type", HitboxInflationType.CN);
```

Comment:
Make the default HitboxInflationType.CTS as per our discussion

Reply:
oh I thought you meant default CN